### DISCUSSION TITLE
Help Needed!

### DISCUSSION THREAD

**Discussion by noobshubham** [oppia/oppia-android#4250] [Q&A (Other)] [answered]
Hey All, Im new to the open source community, I want to contribute to oppia-android. where to start?
I already setted up the project in android studio.

also forgive me  if anything wrong ive done by creating discussion.

**Comment by MaskedCarrot** [comment, marked as answer]
Hi @noobshubham, you can look at [this](https://github.com/oppia/oppia-android/wiki) page to get started.

If you have already signed the CLA and setup oppia-android, you can look at [this](https://github.com/oppia/oppia-android/wiki/Contributing-to-Oppia-android#finding-something-to-do) page to find something to do.
